owasp-zap/README.md

```
@@ -1,38 +1,79 @@
-2) Create a dedicated test user in the StaffUsers Cognito UserPool
-3) Define the following secrets in the GitHub repository with the corresponding information from the staff user pool, client, and test user:
 
-```
-    TEST_COGNITO_USER_POOL_ID_STAFF
-    TEST_WEBROOT_COGNITO_CLIENT_ID_STAFF
-    TEST_ZAP_USERNAME_STAFF
-    TEST_ZAP_PASSWORD_STAFF
-```
+2. **Staff Users pool** — create a dedicated test user with broad scope coverage (e.g. `aslp/admin` plus full `oh` and `ky` jurisdiction permissions). Define these GitHub secrets:
+
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)